airbyte-integrations/connectors/source-unleash/source_unleash/unleash.yaml

```
@@ -0,0 +1,55 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_path: []
+  selector_features:
+    extractor:
+      field_path: ["features"]
+  requester:
+    url_base: "{{ config['api_url'] }}/api/client"
```

This is returning `/api/api/client` for me because the `api_url` I received from Unleash already has /api on the end.

Is there a case where a user would have a url without `/api` on the end? If so, we should support both cases. Otherwise, we can probably remove the `/api` here:

```suggestion
    url_base: "{{ config['api_url'] }}/client"
```